chrome/content/binding.xml

```diff
@@ -12,6 +12,79 @@
         <children/>
       </xul:hbox>
     </content>
+    <implementation>
+      <field name="_selectedIndex">-1</field>
```

Comment:
All this selectedIndex code is needed because we needed to override the existing selectedIndex setter. It's such a convoluted story that I've forgotten exactly why, but I'll piece it together and insert a comment before this lands.